src/AddTodoForm.jsx

```diff
@@ -19,10 +20,9 @@ function AddTodoForm({ onAddTodo }) {
 
   return (
     <div>
-      <form onSubmit={handleAddTodo}> 
-        <label htmlFor="todoTitle">Fill: </label>
-        <input type="text" id="todoTitle" name="todoTitle" value={todoTitle} onChange={handleTitleChange} required/>
-        <button type="submit">Add Todo</button>
+      <form> 
+        <InputWithLabel label="Title:" value={todoTitle} onChange={handleTitleChange} autoFocus={true}/>
```

Comment:
is `label` a property or an attribute? Do we want to pass down a prop here or work with some other function offered by React components?j